src/errors.ts

```diff
@@ -57,3 +57,15 @@ export class UninitializedPurchasesError extends Error {
         Object.setPrototypeOf(this, UninitializedPurchasesError.prototype);
     }
 }
+
+/**
+ * @internal
+ */
+export class UnsupportedPlatformError extends Error {
+    constructor() {
+        super("This method is not available in the current platform.");
+
+        // Set the prototype explicitly.
+        Object.setPrototypeOf(this, UnsupportedPlatformError.prototype);
```

Comment:
Can you explain this?

Reply:
Ah yeah, I forgot to send the comment with my question https://github.com/RevenueCat/react-native-purchases/pull/523#discussion_r1082817543. I've done some tests, and it would seem this is unnecessary. I'm honestly not sure why it's needed, so I asked Cesar there why it was added in the other error (the original PR didn't have any details on this). Depending on that, I might remove it.